CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The following changes have been implemented but not released yet:
 
 ## [Unreleased]
 
+### New features
+
+- `getProfileAll`: functions to discover one's personal profile (which may be a
+  Solid resource) if discoverable from their WebID profile document (which may NOT
+  be a Solid resource).
```

Comment:
```suggestion
- `getProfileAll`: function to discover the WebID Profile Document and its associated
  FOAF Profile Document resources from the WebID URI. Note that Profile documents
  may or not be Solid Resources
```

It is not necessarily "one's" or "personal" (a bot/group/organisation can be identified via a WebID).
All profile documents may or not be a Solid resource (the warning going one way, then the other is confusing to me).

Reply:
PS: also, there is only one function, so either way we need to loose the "s".